Makefile

```
@@ -0,0 +1,34 @@
+DOCKER_USER=saraceni
+THIS_FILE := $(lastword $(MAKEFILE_LIST))
+.PHONY: test postgres-start postgres-stop dev dev-build start stop push-container pull-container
+
+# run tests
+test:
+	go clean -testcache
+	go test ./...
```

Para rodar testes ignorando o cache basta passar `-count=1`.
É bom rodar os testes passando o parametro `-race` para encontrar data races (https://yourbasic.org/golang/detect-data-races/)
```suggestion
	go test -race -count=1 ./...
```